projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 10.8-beta - 2022-03-29
+### Enhancements
+- Add back the upgrade nudge to the Payment Button block.
+- Payment Blocks: Added links to manage subscriptions and fees to the Product Management controls.
+- Adding language attributes to code blocks.
```

Comment:
Since that's documentation, I do not think it will be very interesting for end-users. I would leave it in "Other".